src/app.rs

```diff
@@ -1,22 +1,26 @@
 // #![allow(unused_imports)]
+use rand::{prelude::ThreadRng, thread_rng, Rng};
 use std::{
     env, str, thread,
     time::{Duration, Instant},
 };
 
-use rand::{prelude::ThreadRng, thread_rng, Rng};
+use std::f64;
```

Comment:
Is this needed?

Reply:
> Is this needed?

I needed it for f64::min and f64::max - if you prefer, I can restrict the use to those two functions, or use the fully qualified name instead.

Reply:
I stand corrected - this use is not needed at all. Removed.